filtered-resource-service/src/main/java/uk/gov/gchq/palisade/service/filteredresource/model/WebSocketMessage.java

```diff
@@ -33,17 +34,17 @@
 import java.util.StringJoiner;
 
 /**
- * WebsocketMessage contains the content of the message sent to the websocket and is used by {@link uk.gov.gchq.palisade.service.filteredresource.service.WebsocketEventService}
+ * WebSocketMessage contains the content of the message sent to the webSocket and is used by {@link WebSocketEventService}
  */
-public final class WebsocketMessage {
+public final class WebSocketMessage {
```

Comment:
A few of the Java doc in this class could  be updated with the use of camel case when referring  to WebSocketMessage.

Reply:
Updated Websocket in this class to WebSocket